examples/official-storybook/stories/addon-info.stories.js

```diff
@@ -59,6 +59,14 @@ storiesOf('Addons|Info.Options.inline', module).add(
   })(() => <BaseButton label="Button" />)
 );
 
+storiesOf('Addons|Info.Options.excludedPropTypes').add(
```

Comment:
please don't forget adding second `module` parameter, this is needed for HMR

Reply:
👍Added back in